administrator-manual/en/firewall.rst

```diff
@@ -349,29 +381,44 @@ Each class can have also a maximum rate. If set, the class can exceed its commit
 A class will exceed its committed rate only if there is spare bandwidth available.
 
 Traffic shaping classes can be defined under :guilabel:`Traffic shaping` page.
-When creating a new class, fill in the following fields:
+When creating a new class, fill the following fields.
 
 * :guilabel:`Class name`: a representative name
-* :guilabel:`Min download (%)`: minimum reserved download bandwidth, if empty no download reservation will be created
-* :guilabel:`Max download (%)`: maximum allowed download bandwidth, if empty no upper limit will be set
-* :guilabel:`Min upload (%)`:  minimum reserved upload bandwidth, if empty no upload reservation will be created
-* :guilabel:`Max upload (%)`: maximum allowed download bandwidth, if empty no upper limit will be created
 * :guilabel:`Description`: optional description for the class
 
+Limits under :guilabel:`Download bandwidth limits` section:
+
+  * :guilabel:`Min`: minimum reserved download bandwidth, if empty no download reservation will be created
+  * :guilabel:`Max`: maximum allowed download bandwidth, if empty no upper limit will be set
+
+Limits under :guilabel:`Upload bandwidth limits` section:
+
+  * :guilabel:`Min`:  minimum reserved upload bandwidth, if empty no upload reservation will be created
+  * :guilabel:`Max`: maximum allowed download bandwidth, if empty no upper limit will be created
+
+
+For each class the bandwidth can be specified using the percentage of available network bandwidth or
+with absolutes values expressed in kbps.
```

Comment:
```suggestion
with absolutes values expressed in Kbps.
```

Reply:
The right spelling is with lower case: https://en.wikipedia.org/wiki/Kilobit